CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ and this project attempts to adhere to Semantic Versioning.
 
 ## UNRELEASED
 
+### Added
+
+* `archiveWith`, `archiveTarget`, `batchId`, `stopProcessing` fields for case archive
+  * if a case archive is resubmitted with changed data and the case has already begun archiving, the case will be set to `stopProcessing` and monitoring will be updated
+* `/case/<caseIdentifier>/resume-archiving` endpoint to manually resume archiving if a case had been set to `stopProcessing`
```

Comment:
When would we or wouldn't we resume archiving if stopProcessing is set?

Reply:
If the mismatching data that caused stopProcessing to be set was determined to be an error and was resolved. For instance, the archiving olive recently created case archive actions with workflowRunsForOnsiteArchive containing both clinical and archival workflow run IDs - the olive got fixed to not include archival workflow run IDs for that field (since archiving from vidarr-archival to vidarr-archival doesn't make sense) and is now generating actions with the same data as is already in Nabu. So , what do you think about:

* `/case/<caseIdentifier>/resume-archiving` endpoint to manually resume archiving if a data mismatch had caused the case to be set to `stopProcessing` and the data mismatch had been resolved